x-pack/plugins/maps/server/routes.js

```diff
@@ -598,6 +600,40 @@ export async function initRoutes(
   );
 
   if (drawingFeatureEnabled) {
+    router.post(
+      {
+        path: `/${GIS_API_PATH}/feature`,
+        validate: {
+          body: schema.object({
+            index: schema.string(),
+            data: schema.any(),
```

Comment:
Can we make this stricter than `any`? This should at least be an object with key-value pairs I think.

Reply:
We know the schema for coordinates, but the shape of the data beyond that is murky since it's entirely dependant on the mappings. Mappings and the data ideally need to stay open ended for now to handle adding to new or existing indexes which might differ in their shapes. We could eventually be more prescriptive with what mappings/data-shapes we accept, but personally I'd prefer to wait until we have full roundtrip drawing functionality in place before we start tightening options.